src/hooks/postMessage.ts

```diff
@@ -156,15 +182,28 @@ const configurePostMessageHooks = (_queryClient: QueryClient, queryConfig: Query
         }
       },
     });
+  };
 
-  let getAuthTokenFunction;
-  const useAuthToken = (itemId: string) =>
-    useQuery({
+  const useAuthToken = (itemId: string): UseQueryResult<string> => {
+    let getAuthTokenFunction;
+    const queryClient = useQueryClient();
+    return useQuery({
       queryKey: AUTH_TOKEN_KEY,
       queryFn: () => {
+        // eslint-disable-next-line no-console
+        console.debug('[apps-query-client] Get Auth token');
+        if (queryConfig.isStandalone) {
+          const context = queryClient.getQueryData<LocalContext>(LOCAL_CONTEXT_KEY);
+          if (context) {
+            return `${MOCK_TOKEN} ${context.memberId}`;
+          } else {
+            throw new Error('there was an error getting the query data for the LocalContext');
+          }
+        }
         const POST_MESSAGE_KEYS = buildPostMessageKeys(itemId);
         if (!port2) {
           const error = new MissingMessageChannelPortError();
+          // eslint-disable-next-line no-console
```

Comment:
This rule seems cumbersome. Maybe we want to warn for log but not for error and debug?